components/bl-signature-pad-component/src/index.js

```diff
@@ -0,0 +1,54 @@
+import ActionButtons from './actionButtons.js';
+import SignaturePad from './leap/signature_pad.umd.min.js';
+
+export default function MyCustomComponent({ component, eventHandlers }) {
+  const elRef = React.useRef(null);
+  const signaturePad = React.useRef(null);
+  
+  function resizeCanvas() {
+    const ratio = Math.max(window.devicePixelRatio || 1, 1);
+    const canvas = elRef.current;
+    canvas.width = canvas.offsetWidth * ratio;
+    canvas.height = canvas.offsetHeight * ratio;
+    canvas.getContext("2d").scale(ratio, ratio);
+    signaturePad.current.clear();
+  }
+
+  React.useEffect(() => {
+    signaturePad.current = new SignaturePad(elRef.current, {
+      backgroundColor: 'rgb(255, 255, 255)',
```

Comment:
is it possible to specify these styles in Less?

Reply:
No, it's impossible. I can specify the background color as is or pass it through properties.

Reply:
does it support `inherit` value, if so, we can set `backgroundColor: inherit ` and specify the color for the container(wrapper)

Reply:
It works by default after I added the background to the container and removed it from the SignaturePad. Thanks

Reply:
I was wrong. The default is "rgba (0,0,0,0)" (transparent black). 
Also, it doesn't support inherit value.